apache/incubator-zeppelin
# [ZEPPELIN-819] Add 'Last updated by X at Y' message in paragraph footer

### What is this PR for?
In a large organization, it is helpful to know who updated a paragraph along with the time.

### What type of PR is it?
Improvement (Minor)

### Todos
* [x] - Understand why the user is not updated immediately after running the paragraph.

### What is the Jira issue?
[ZEPPELIN-819] (https://issues.apache.org/jira/browse/ZEPPELIN-819)

### How should this be tested?
1. Login to the zeppelin server
2. Create a note
3. Run a paragraph
4. Reload

Paragraph footer should display the 'Last updated by X at Y' message

### Screenshots (if appropriate)
NA

### Questions:
NA
